backend/src/main/java/ai/giskard/domain/ml/Slice.java

```diff
@@ -0,0 +1,30 @@
+package ai.giskard.domain.ml;
+
+import ai.giskard.domain.AbstractAuditingEntity;
+import ai.giskard.domain.Project;
+import com.fasterxml.jackson.annotation.JsonBackReference;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+
+import javax.persistence.Entity;
+import javax.persistence.ManyToOne;
+import javax.validation.constraints.NotNull;
+
+@Entity(name = "slices")
+@NoArgsConstructor
+@Getter
+@Setter
+public class Slice extends AbstractAuditingEntity {
+    @NotNull
+    private String name;
+
+    // Later on this will probably depend on the slice type: From a file, from the interface, from the API ?
+
+    @NotNull
+    private String code;
+
+    @ManyToOne
+    @NotNull
+    private Project project;
```

Comment:
There needs to be a back reference in Project.java with `cascade = CascadeType.ALL` so that the project deletion works

Reply:
like for ai.giskard.domain.Project#testSuites